### DISCUSSION TITLE
Page does not exists

### DISCUSSION THREAD

**Discussion by numfin** [framesurge/perseus#253] [General]
I'm reading first page of perseus book and there is 404 for
https://framesurge.sh/perseus/en-US/docs/0.3.4/reference/strategies/incremental-generation

**Comment by arctic-hen7** [comment]
Ah sorry, that should be https://framesurge.sh/perseus/en-US/docs/0.3.4/reference/strategies/incremental.

I'll update that link right away, thanks for letting me know!
